### PR TITLE
- Improve PipeOpFilter docu

### PR DESCRIPTION
**Pull request by sebffischer**

### CODE REVIEW

File: R/FilterMRMR.R [hunk 1]
```diff
@@ -19,10 +19,24 @@
...
+#'   filter = flt("mrmr")
+#'   filter$calculate(task, nfeat = 2)
+#'   as.data.table(filter)
+#' }
+#'
+#' if (mlr3misc::require_namespaces(c("mlr3pipelines", "MASS", "praznik"), quietly = TRUE)) {
+#'   library("mlr3pipelines")
+#'   task = mlr3::tsk("spam")
+#'
+#'   # Note: The filter.frac is selected randomly and should be tuned.
```
Comment (pat-s) [inline R/FilterMRMR.R:33]: You can leave the "the" out here and in all other occasions, just use `filter.frac` (formatting the argument as code would also be nice).

File: R/FilterNJMIM.R [hunk 2]
```diff
@@ -19,10 +19,24 @@
...
-#' filter$calculate(task, nfeat = 2)
-#' as.data.table(filter)
+#' if (requireNamespace("praznik")) {
+#'   task = mlr3::tsk("iris")
+#'   filter = flt("njmim")
+#'   filter$calculate(task, nfeat = 2)
+#'   as.data.table(filter)
+#' }
+#'
+#' if (mlr3misc::require_namespaces(c("mlr3pipelines", "MASS", "praznik"), quietly = TRUE)) {
```
Comment (pat-s) [inline R/FilterNJMIM.R:29]: What is exactly is {MASS} needed for in all the examples? Do we only need it in some?

File: README.md [hunk 3]
```diff
@@ -65,38 +65,41 @@ as.data.table(filter$calculate(task))
...
-| njmim             | Classif & Regr | Integer, Numeric, Factor, Ordered          | [praznik](https://cran.r-project.org/package=praznik)             |
-| performance       | Universal      | Logical, Integer, Numeric, Factor, Ordered |                                                                   |
-| permutation       | Universal      | Logical, Integer, Numeric, Factor, Ordered |                                                                   |
-| relief            | Classif & Regr | Integer, Numeric, Factor, Ordered          | [FSelectorRcpp](https://cran.r-project.org/package=FSelectorRcpp) |
-| selected_features | Classif        | Logical, Integer, Numeric, Factor, Ordered |                                                                   |
-| variance          | Classif & Regr | Integer, Numeric                           | stats                                                             |
+    ## Warning in `[.data.table`(as.data.table(mlr_filters), , !c("param_set", :
+    ## column(s) not removed because not found: [param_set]
```
Comment (pat-s) [inline README.md:69]: Note this warning

File: README.md [hunk 4]
```diff
@@ -131,3 +134,24 @@ head(as.data.table(filter), 3)
 final outcome using the supplied measure. Any learner can be passed to
 this filter with `classif.rpart` being the default. Of course, also
 regression learners can be passed if the task is of type “regr”.
+
+### Filter-based Feature Selection
+
+In many cases filtering is only one step in the modeling pipeline. To
+select features based on filter values, one can use
+[`PipeOpFilter`](https://mlr3pipelines.mlr-org.com/reference/mlr_pipeops_filter.html)
+from [mlr3pipelines](https://github.com/mlr-org/mlr3pipelines).
```
Comment (pat-s) [inline README.md:143]: One sentence per line please :) (maybe worth checking your editor settings, it seems line breaks are enforced at 80 chrs?)

**Review by pat-s — CHANGES_REQUESTED** [review]
Thanks! I merged Michels PR before and now there are some conflicts but I guess you should be able to handle them 🙂

File: README.md (hunk 4 above)
Comment (sebffischer) [reply README.md:143]: Note that this is the md file generated from the Rmd

File: R/FilterNJMIM.R (hunk 2 above)
Comment (sebffischer) [reply R/FilterNJMIM.R:29]: Ah sorry, I somehow though rpart was in MASS but it isn't.

File: R/FilterMRMR.R (hunk 1 above)
Comment (sebffischer) [reply R/FilterMRMR.R:33]: done

**Comment by sebffischer** [comment]
for some reason I cannot build the README.html because of a http error

**Review by pat-s — APPROVED** [review]
Thanks Seb! 👍

**Pull request merged**